src/views/BottomTabBar.js

```diff
@@ -168,7 +168,28 @@ class TabBarBottom extends React.Component<Props> {
 
   _shouldUseHorizontalLabels = () => {
     const { routes } = this.props.navigation.state;
-    const { isLandscape, dimensions, adaptive, tabStyle } = this.props;
+    const {
+      isLandscape,
+      dimensions,
+      adaptive,
+      tabStyle,
+      tabDirection,
+    } = this.props;
+
+    if (tabDirection) {
+      let direction;
+      if (typeof direction === 'string') {
```

Comment:
this will never be a string

Reply:
Err, this is supposed to be `tabDirection`. I'll check later but this may not be the latest commit :(. My bad

Reply:
updated